models/fetchReviews.model.js

```diff
@@ -0,0 +1,12 @@
+const db = require('../db/connection');
+
+function fetchReviews() {
+    return db.query(`SELECT reviews.*, CAST(COUNT(comments.review_id) AS INT) AS comment_count FROM reviews LEFT JOIN comments on reviews.review_id = comments.review_id GROUP BY reviews.review_id ORDER BY reviews.created_at DESC;`).then((review) => {
+        if (review.rows.length === 0) {
+            return Promise.reject({ status: 404, msg: "ID does not exist, please use a valid ID" })
+        }
```

Comment:
Not sure what this is doing, as there isnt an id in this endpoint. 
I think it wouldnt make any difference if you took this out, i dont see a test it would fail.